_rules/object-has-acessible-name-8fc3b6.md

```diff
@@ -156,16 +155,26 @@ This `object` element is not [included in the accessibility tree][] due to `aria
 This `object` element is not [included in the accessibility tree][] because it is marked as decorative through `role="presentation"`.
 
 ```html
-<object type="image/png" role="presentation" data="/test-assets/contrast/example.png"> </object>
+<object type="image/png" role="presentation" data="/test-assets/contrast/example.png"></object>
 ```
 
 #### Inapplicable Example 5
 
+This `object` element embeds an HTML resource.
+
+```html
+<object title="My University" data="/test-assets/shared/index.html"></object>
```

Comment:
This unusual scenario feels similar to an iframe. Just double checking, but presume for this rule/example it doesn't matter if the HTML resource is non-text content?

Reply:
It is similar to an iframe (I seem to remember one of the browsers even treating this as an iframe in the DOM).
The rule was revised to be applicable only to images, audio or video mime types. That's why this was moved to an Inapplicable example. 
Do let me know if you wish to change your approval!